app/views.py

```diff
@@ -58,4 +58,19 @@ def register():
 @app.route('/trips')
 @login_required
 def trips():
-    return render_template('trips.html')
\ No newline at end of file
+    return render_template('trips.html')
+
+
+
+@app.route('/trips/create', methods=['GET', 'POST'])
+@login_required
+def create_trip():
+    form = TripForm()
+    if form.validate_on_submit():
+        trip = trip(name=form.name.data, destination=form.destination.data)

```

Comment:
What you want to do here is to create a new variable called `trip` (note the lowercase spelling) which is of type `Trip` (you have to import it from app.models).

So it should be
```
 trip = Trip(name=form.name.data, destination=form.destination.data)
```